cvmfs/cvmfs_server

```diff
@@ -34,7 +34,7 @@ CVMFS_UPDATEGEO_HOUR=10 # First hour of day for update, 0-23, default 10am
 CVMFS_UPDATEGEO_MINDAYS=25 # Minimum days between update attempts
 CVMFS_UPDATEGEO_MAXDAYS=100 # Maximum days old before considering it an error
 
-CVMFS_UPDATEGEO_URLBASE="http://geolite.maxmind.com/download/geoip/database"
+CVMFS_UPDATEGEO_URLBASE="${CVMFS_UPDATEGEO_URLBASE:=http://geolite.maxmind.com/download/geoip/database}"
 CVMFS_UPDATEGEO_URLBASE6="${CVMFS_UPDATEGEO_URLBASE}/GeoLiteCityv6-beta"
```

Comment:
Thanks for the valid comments. Since I use the same variable (i.e. `CVMFS_UPDATEGEO_URLBASE`) the ":=" shouldn't do any harm here. Though I wasn't aware of this side effect.

Anyway, I'll have a look if I can use `cvmfs_server_hooks.sh` for the same purpose.